tasks/hana_deploy.yml

```diff
@@ -19,6 +19,23 @@
     - not(( sap_hana_deployment_bundle_path is none ) or ( sap_hana_deployment_bundle_path | trim == ''))
     - not(( sap_hana_deployment_bundle_sar_file_name is none ) or (sap_hana_deployment_bundle_sar_file_name | trim == ''))
 
+- name: Unzip ZIP archive 
```

Comment:
Please try to use `unarchive` Ansible module for this instead `command`

Reply:
Done.